api/src/test/java/uk/gov/dhsc/htbhf/claimant/entity/ClaimantTest.java

```diff
@@ -298,4 +299,25 @@ void shouldValidateClaimantWithValidEmailAddress(String emailAddress) {
         //Then
         assertThat(violations).hasNoViolations();
     }
+
```

Comment:
Can we have an empty list test as well.

Reply:
The first test does that by using `aValidClaimant()` which has no children's date of births in it.